patterns/2-structured/support-as-a-group.md

```diff
@@ -0,0 +1,86 @@
+## Title
+
+Support as a Group
+
+## Patlet
+
+What happens when the team or individual that was supporting an InnerSource project is no longer able to do so?
+Form a group of interested individuals from anywhere in the company to keep the project going.
+
+## Problem
+
+* A popular InnerSource project is orphaned with no one to support it.
+* There isn't an obvious home for it to land.
+
+## Story
+
+A library of UI widgets is used by over 50 projects throughout the company.
+The funding for the team that owns the library runs out and the team disbands.
+At first, no one notices, but after a while whenever someone asks "who owns it" there isn't an answer.
+What will happen next?
+Will new teams shy away from using it?
+Will the project stagnate and linger until its users eventually are forced to move on to something else?
+What a shame if that were to happen to a perfectly good and useful project!
+
+## Context
+
+* Popular InnerSource project.
+* No one is actively supporting it.
```

Comment:
```suggestion
* No one is actively supporting it.
* The company cannot assign a new maintainer to the project because ...
```

Suggesting to add the reason for not assigning a new maintainer here, or maybe a list of possible reasons.

This bit got me confused, as it didn't seem logical to me.
The library has a lot of users (50 projects).
The migration cost away from this library is high.
50 x high migration cost = super high cost :)

So why wouldn't the company assign a new maintainer?
Wouldn't it be in their best interest?

It is entirely possible that I am misunderstanding the scenario/context here.
And if yes that's great, so that we can clarify this :)

Reply:
I see. Still confused by this, I have to admit :)

Reply:
The company is probably short-staffed in general.